<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTextureWnd</class>
 <widget class="QDialog" name="AddTextureWnd">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MetroTEX - Add texture</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Please choose how do you want to add a texture:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkUseExistingTexture">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Existing converted texture:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpAddExistingTexture">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>651</width>
     <height>40</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="txtExistingTexturePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowseExistingTexture">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>651</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkAddAndConvertTexture">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Add and convert from source image:</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpAddAndConvertTexture">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>651</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Path to source texture:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAddAndConvertSourcePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowseSourceTexture">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>29</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output folder path:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAddAndConvertDestPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowseOutputFolder">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>89</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddTextureWnd</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddTextureWnd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
